plugins/network/NetCfg.cpp

```diff
@@ -52,16 +53,17 @@ int Config()
 		switch (HiddenSharesState)
 		{
 		case 0: // never show
-			Opt.HiddenShares = FALSE;
-			Opt.HiddenSharesAsHidden = FALSE;
-			break;
-		case 1: // make hidden
-			Opt.HiddenShares = TRUE;
-			Opt.HiddenSharesAsHidden = TRUE;
+			Opt.HiddenShares = false;
+			Opt.HiddenSharesAsHidden = false;
 			break;
 		case 2: // always show
-			Opt.HiddenShares = TRUE;
-			Opt.HiddenSharesAsHidden = FALSE;
+			Opt.HiddenShares = true;
+			Opt.HiddenSharesAsHidden = false;
+			break;
+		case 1: // make hidden
+		default:
```

Comment:
Why do we need this change?
The value comes from a dialog, we won't suddenly get `42` here.

Reply:
>we won't suddenly get 42 here.
sure ? ;) ok, even if this happens, then the cost of error here is small